source/Calamari.AzureAppService/Behaviors/AzureAppServiceBehaviour.cs

```diff
@@ -5,6 +5,7 @@
 using System.Linq;
 using System.Net.Http;
 using System.Net.Http.Headers;
+using System.Runtime.CompilerServices;
```

Comment:
Is this required?

Reply:
probably not, not sure where that came from